packages/react-magma-dom/src/components/IndeterminateCheckbox/index.tsx

```diff
@@ -159,37 +170,45 @@ export const IndeterminateCheckbox = React.forwardRef<
           ref={ref}
           type="checkbox"
           onChange={handleChange}
+          tabIndex={-1}
         />
-        <StyledLabel htmlFor={id} isInverse={isInverse} style={labelStyle}>
-          <StyledFakeInput
-            isChecked={isChecked}
-            color={color}
-            disabled={disabled}
-            hasError={hasError}
-            hideFocus={props.hideFocus}
-            isIndeterminate={isIndeterminate}
-            isInverse={isInverse}
-            style={inputStyle}
-            theme={theme}
-            aria-hidden="true"
-          >
-            {isIndeterminate ? (
-              <IndeterminateCheckBoxIcon
-                testId="indeterminateIcon"
-                size={theme.iconSizes.medium}
-              />
-            ) : isChecked ? (
-              <CheckBoxIcon size={theme.iconSizes.medium} />
+        <div
+          tabIndex={!disabled && 0}
```

Comment:
I'm not sure about this tabindex here. It's changing the keyboard navigation behavior in TreeView which we don't want. Speaking of which, we should note in the testing notes to consider TreeView and DataGrid

Reply:
I removed the div and moved these attributes to `StyledFakeInput`.